config/develop/example-dev-project.j2

```diff
@@ -1,28 +1,40 @@
-stack_name: example-dev-project
+{% set stack_name = "example-dev-project" %}
+{% set read_write_users = ["bruno.grande@sagebase.org"] %}
+{% set read_only_users = ["tess.thyer@sagebase.org", "thomas.yu@sagebase.org"] %}
 
-stack_tags:
-  Department: IBC
-  Project: Infrastructure
-  OwnerEmail: nextflow-admins@sagebase.org
+# DO NOT EDIT ANYTHING BELOW THIS LINE -----------------------------------------------------------
+
+template_path: tower-project.yaml
+stack_name: {{ stack_name }}
+dependencies:
+  - common/nextflow-forge-iam-policy.yaml
+  - common/nextflow-launch-iam-policy.yaml
 
 parameters:
+  {% if read_write_users %}
   S3ReadWriteAccessArns:
-    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/bruno.grande@sagebase.org
+  {% for user in read_write_users %}
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/{{ user }}
+  {% endfor %}
+  {% endif %}
+  {% if read_only_users %}
   S3ReadOnlyAccessArns:
-    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/tess.thyer@sagebase.org
-
-  # DO NOT EDIT ANYTHING BELOW THIS LINE ------------------------------------------------------------------------
-
+  {% for user in read_only_users %}
+    - arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/{{ user }}
+  {% endfor %}
+  {% endif %}
   AccountAdminArns:
     - {{stack_group_config.sso_admin_role.arn}}
     - !stack_output_external workflows-nextflow-ci-service-account::ServiceRoleArn
   TemplateRootUrl: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com'
   TowerForgePolicyArn: !stack_output_external nextflow-forge-iam-policy::NextFlowForgePolicyArn
   TowerLaunchPolicyArn: !stack_output_external nextflow-launch-iam-policy::NextFlowLaunchPolicyArn
 
-template_path: tower-project.yaml
+stack_tags:
+  Department: IBC
+  Project: Infrastructure
+  OwnerEmail: nextflow-admins@sagebase.org
```

Comment:
I like the idea of consolidating all production Tower project into a new folder like `config/projects/`. This would also make it easier for the CI step to find these stack config files and parse them to configure Tower.

Reply:
> Maybe we should use a jinja include for this.
Are we okay with making all of our infra configs into Jinja templates for this? Alternatively, we can add the values into the stack group config and refer to them using `{{stack_group_config.infra_department}}`, `{{stack_group_config.infra_project}}` and `{{stack_group_config.infra_email}}`.

Reply:
We don't have to do this, I just find it slightly annoying to have this duplicated everywhere and wanted to replace three lines with one. Let's leave it for now. We have bigger fish in need of frying.

Reply:
I made note of your conversation in the sageit Slack channel.

Reply:
> Are we okay with making all of our infra configs into Jinja templates for this? Alternatively, we can add the values into the stack group config and refer to them using `{{stack_group_config.infra_department}}`, `{{stack_group_config.infra_project}}` and `{{stack_group_config.infra_email}}`.

you could just create a dict for department, project and email in the top level config.yaml file and reference it like this..
```
stack_tags:
  {{stack_group_config.stack_tags}}
```

Reply:
@zaro0508: Good idea! I created a JIRA ticket for carrying out the chance to avoid bloating this PR any further. 

https://sagebionetworks.jira.com/browse/WORKFLOWS-62

Reply:
+1 that works thanks @zaro0508